lib/rules/require-unicode-regexp.js

```diff
@@ -37,36 +37,102 @@ module.exports = {
 
         messages: {
             addUFlag: "Add the 'u' flag.",
-            requireUFlag: "Use the 'u' flag."
+            requireUFlag: "Use the 'u' flag.",
+            addVFlag: "Add the 'v' flag.",
+            requireVFlag: "Use the 'v' flag."
         },
 
-        schema: []
+        schema: [{
+            type: "object",
+            properties: {
+                requiredUnicodeFlag: {
+                    type: "string",
+                    enum: ["u", "v"]
```

Comment:
I don't think completely disallowing the `v` flag (by requiring the `u` flag) makes sense for this rule.

For example:

```js
/* eslint require-unicode-regexp: ["error", { requiredUnicodeFlag: "u" }] */

const regex = /[[a-z]--[j-n]]/v; // error: Use the 'u' flag.

```

In this regex, the `v` flag cannot be simply replaced with the `u` flag, so this rule would be basically disallowing using the new syntax in patterns.

I think, for the start, the rule should just allow regular expressions with the `v` flag, and maybe have an additional suggestion to add the `v` flag for those that have neither `u` nor `v`. Later, if there is a request, we could consider adding options to enforce a particular flag.

Reply:
Thank you for reviewing. I've addressed your review!